crates/c/src/lib.rs

```diff
@@ -1091,6 +1108,24 @@ void __wasm_export_{ns}_{snake}_dtor({ns}_{snake}_t* arg) {{
     }
 }
 
+impl<'a> InterfaceGenerator<'a> {
+    fn reosurce_handle_names(&mut self, id: TypeId, name: &str) -> (String, String) {
```

Comment:
Is there a way to get `TypeId` of handles given that of a Resource?

Reply:
This is not, no